src/hooks/useDimensions.ts

```diff
@@ -30,6 +31,7 @@ export interface DeviceDimensions extends ScaledSize {
 
 export default function useDimensions(): DeviceDimensions {
   const { height, width, ...restOfDimensions } = useWindowDimensions();
+  analytics.identify(undefined, { screenWidth: width, screenHeight: height });
```

Comment:
how do you feel about moving this to `handleInitializeAnalytics` in App.js maybe? this hook is called many times from different parts of the app and we'd be calling analytics multiple times because of that

Reply:
good point, just moved